modules/new_serverpod_auth/serverpod_auth_jwt/serverpod_auth_jwt_server/lib/src/models/refresh_token.spy.yaml

```
@@ -0,0 +1,58 @@
+class: RefreshToken
+serverOnly: true
+table: serverpod_auth_jwt_refresh_token
+fields:
+  id: UuidValue?, defaultPersist=random
+
+  ### The [AuthUser] this profile belongs to.
+  authUser: module:auth_user:AuthUser?, relation(onDelete=Cascade)
+
+  ### The scopes given to this session.
+  ###
+  ### These will also be added to each access token as a claim named "scopeNames".
+  scopeNames: Set<String>
+
+  ### Extra claims to be added to each access token created for this refresh token.
+  ###
+  ### This is a `Map<String, dynamic>` where each entry uses the key as the claim name.
+  ### The value must be convertible to JSON.
```

Rename (locally + gen)

```suggestion
  ### The values must be JSON-encodable.
```